Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -138,10 +150,14 @@ export default class CovidTreatmentAssessmentComponent extends Vue {
     }
 
     private get patientFullName(): string {
+        this.covidTreatmentAssessmentRequest.firstName = this.patient.firstname;
+        this.covidTreatmentAssessmentRequest.lastName = this.patient.lastname;
         return `${this.patient.firstname} ${this.patient.lastname} `;
     }
 
     private get patientPersonalHealthNumber(): string {
+        this.covidTreatmentAssessmentRequest.phn =
+            this.patient.personalhealthnumber;
```

Comment:
Same as above.  We shouldn't have side effects in getters; this value can be set right before we submit the request.

Reply:
@MikeLyttle but there is no issue also if we create a separate function?

Reply:
See above.